OxfordBodleian/Juel-Jensen/BDLaethb6.xml

```diff
@@ -9153,12 +9157,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
-                                    <p>Small tears, presumably resulting from parchment making, carefully amended on very few folia, e.g. <locus target="#21 #73 #162"/>.</p>
-                                    <p>The small rounded holes presumably from parchment making, on <locus target="#41 #68 #155"/> have not been repaired.</p>
+                                    <p>Small tears resulting from parchment making carefully amended on very few folia, e.g. 
```

Comment:
Corrected here.

Reply:
You are right, we had agreed to substitute leaf by folio. I would propose to do this everywhere, but I have two questions:

1. In the manuscripts consisting of a single leaf, such as BDLjj29, we have sentences as "The leaf is not foliated" or "the leaf had been inserted into `<ref type="mss" corresp="BDLaethb3"/>`,". My feeling would be that substituting here by "folio" would lead to strange sounding sentences. What do you think?
2. In nearly all manuscripts, we have in dimensions the sentence "Data on leaves dimensions taken from..." - is it also fine to change this to "Date on the dimensions of folia taken from..."?

Reply:
I agree that in 1 changing from leaf to folio it would result in a strange sounding sentence. As regards the point 2, if the sentence is used in every manuscript description it would make sense to keep it, but changing to "Date on the dimensions of folia taken from..." is also fine, it depends how much time consuming it is.

Reply:
Concerning 1., "the folio is unfoliated" (and similar) sounds indeed weird. But if we decide to change "leaf" > "folio" and "leaves" > "folios", we could rephrase it like this: "the unit/manuscript is unfoliated" or simply "Unfoliated."
But I do not make it a fundamental point. If the two terms are absolutely equivalent and an alternation between the two is not confusing, both can be left